manifests/overlays/prod/applications/aiops/aiops-prod-argo.yaml

```diff
@@ -8,9 +8,9 @@ spec:
     server: https://api.ocp.prod.psi.redhat.com:6443
   project: aiops
   source:
-    path: applications/argo/overlays/aiops-prod-argo
```

Comment:
This path didn't exist in the source repository either, so I'm not sure here..

Reply:
it's fine, it'll complain but that's expected for now